vendor/github.com/form3tech-oss/jwt-go/claims.go

```diff
@@ -0,0 +1,136 @@
+package jwt
+
+import (
+	"crypto/subtle"
+	"fmt"
+	"time"
+)
+
+// For a type to be a Claims object, it must just have a Valid method that determines
+// if the token is invalid for any supported reason
+type Claims interface {
+	Valid() error
+}
+
+// Structured version of Claims Section, as referenced at
+// https://tools.ietf.org/html/rfc7519#section-4.1
+// See examples for how to use this with your own claim types
+type StandardClaims struct {
+	Audience  []string `json:"aud,omitempty"`
```

Comment:
this changed the type from `string` to `[]string`... will that break compatibility with decoding any existing tokens?

Reply:
If Azure happily changed it in their codebase, I assumed things would be fine. :man_shrugging:.